browser/src/control/Control.Sidebar.ts

```diff
@@ -9,44 +9,59 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 /*
- * L.Control.Sidebar
+ * Control.Sidebar
  */
 
-/* global $ app */
-L.Control.Sidebar = L.Control.extend({
+/* global app */
+interface SidebarOptions {
+	animSpeed: number;
+}
+class Sidebar extends L.Control {
```

Comment:
no need for `extends L.Control` we want to completely drop Leaflet's dependency

Reply:
it will be needed to also modify where we initialize the component then,
instead of `map.addControl(sidebar)`, we need to create sidebar object and then call `onAdd` method manually, or put it inside constructor.